lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -10,6 +10,9 @@ class AnsibleRunner < ::Proxy::Dynflow::Runner::Parent
       include ::Proxy::Dynflow::Runner::ProcessManagerCommand
       attr_reader :execution_timeout_interval
 
+      # To make this overridable in development
+      ENVIRONMENT_WRAPPER = ENV['SMART_PROXY_ANSIBLE_ENVIRONMENT_WRAPPER'] || '/usr/libexec/foreman-proxy/ansible-runner-environment'
```

Comment:
This is missing the `.sh` file extension

Reply:
It really shouldn't ;)
https://github.com/theforeman/smart_proxy_ansible/pull/66#discussion_r958544437